npm_and_yarn/spec/fixtures/github/package_json_with_nested_glob_workspaces.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "package.json",
+  "path": "package.json",
+  "sha": "5c7b3419e0056515122b981f1566ebe22c208251",
+  "size": 594,
+  "url": "https://api.github.com/repos/gocardless/bump/contents/package.json?ref=master",
+  "html_url": "https://github.com/gocardless/bump/blob/master/package.json",
+  "git_url": "https://api.github.com/repos/gocardless/bump/git/blobs/5c7b3419e0056515122b981f1566ebe22c208251",
+  "download_url": "https://raw.githubusercontent.com/gocardless/bump/master/package.json?token=ABMwe0apDiKCctWHnEHnszRBAebVHjQnks5WJWD9wA%3D%3D",
```

Comment:
This just returns `{}` for me. Maybe we can hardcode it?

Reply:
hmmm, I believe I've just copied that from another fixture, and all it really does is try to mimic what the "real" response would look like. But other than that it's just some fake data, I guess we shouldn't care what it actually returns.